sharing-service/api-config.js

```diff
@@ -0,0 +1,53 @@
+module.exports = serverless => {
```

Comment:
`api-config` should not be required in `sharing-services`. From what I can see, it's not used.